cmd/manager/main.go

```diff
@@ -230,6 +226,13 @@ func ensureDefaults(cfg *rest.Config, scheme *kruntime.Scheme) error {
 
 func GenerateMetrics(cfg *rest.Config, namespace string) {
 
+	// convert the value of MetricsPort from string to int32.
+	port, err := strconv.ParseInt(customMetrics.MetricsPort, 10, 32)
```

Comment:
Why not just defined `customMetrics.MetricsPort` as an `int32`?

Reply:
Because `http.ListenAndServe` requires a string.